library/proc_macro/src/lib.rs

```diff
@@ -1200,3 +1200,17 @@ pub mod tracked_env {
         value
     }
 }
+
+/// Tracked access to additional files.
+#[unstable(feature = "track_path", issue = "73921")]
```

Comment:
I think it'd be good to split the currently introduced feature and use a dedicated `proc_macro_tracked_path` for parity with `proc_macro_tracked_env`